downstream/modules/platform/proc-configure-controller-OCP.adoc

```diff
@@ -0,0 +1,106 @@
+[id="configure-controller-OCP"]
+
+During a Kubernetes upgrade, {ControllerName} has to be running. The following procedure outlines the configuration changes you need to make in automation controller to minimize downtime during the upgrade.
```

Comment:
this also help with any "voluntary disruption"

https://kubernetes.io/docs/concepts/workloads/pods/disruptions/#voluntary-and-involuntary-disruptions

this shows more information about what voluntary disruptions are in kubernetes